### PR TITLE
chore: display new version on update message

### PR DESCRIPTION
**Pull request by PatrickMenoti**

### CODE REVIEW

**Review by gabriel-azion — APPROVED** [review]

**Review by maxwelbm — APPROVED** [review]

File: pkg/cmd/root/sysinfo_darwin.go [hunk 1]
```diff
@@ -3,12 +3,14 @@
 package root
 
 import (
+	"fmt"
+
```
Comment (PabloDiehl) [inline pkg/cmd/root/sysinfo_darwin.go:7]: I dunno if it's a Go pattern, but do we need this empty line here?

File: pkg/cmd/root/sysinfo_freebsd.go [hunk 2]
```diff
@@ -3,13 +3,15 @@
 package root
 
 import (
+	"fmt"
+
```
Comment (PabloDiehl) [inline pkg/cmd/root/sysinfo_freebsd.go:7]: And here?

**Review by PabloDiehl — APPROVED** [review]

File: pkg/cmd/root/sysinfo_darwin.go (hunk 1 above)
Comment (PatrickMenoti) [reply pkg/cmd/root/sysinfo_darwin.go:7]: Go linter automatically puts the new line

File: pkg/cmd/root/sysinfo_freebsd.go (hunk 2 above)
Comment (PatrickMenoti) [reply pkg/cmd/root/sysinfo_freebsd.go:7]: Go linter automatically puts the new line

**Pull request merged**